### DISCUSSION TITLE
Quit guard operates on all sheets, not just modified ones.

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1848] [Q&A]
The [manpage](https://www.visidata.org/man/) says:

    --quitguard                  False              confirm before quitting modified sheet

However, when I enable the guard, it warns about closing *all* sheets, not just the modified ones. Also, should it apply to modified *and* new sheets? (Is new considered modified?)

**Comment by saulpw** [comment]
Hm, I think that's a bug then.  It should only warn for modified sheets.

**Reply by reagle** [reply]
Let me know if you can't confirm, I'll then dig more and see what might be going on here. (I did reproduce it on two, similar, computers: macOS 13.3.1 22E261 arm64 with vd v2.11.)
